zp-relayer/index.ts

```diff
@@ -2,13 +2,14 @@ import './env'
 import express from 'express'
 import router from './router'
 import { logger } from './services/appLogger'
-import { createLoggerMiddleware } from './services/loggerMiddleware'
+import { createConsoleLoggerMiddleware, createPersistentLoggerMiddleware } from './services/loggerMiddleware'
 import config from './config'
 import { init } from './init'
 
 const app = express()
 
-app.use(createLoggerMiddleware('zp.log'))
+app.use(createPersistentLoggerMiddleware('zp.log'))
```

Comment:
May we make the path to the directory where the file is located configurable as so the corresponding directory can be mounted in the docker environment? In this case there will be no need to get the access to the file within the container.